### ISSUE TITLE
Grafana Image Renderer plugin prevents systemd service from starting correctly

### ISSUE THREAD

**Issue by Darktortue** [grafana/grafana#42531]
**What happened**:
Upgraded my Grafana systemd service from 7.5.11 to to 8.2.5 with the usuals `apt` commands and then installed the Grafana Image Renderer plugin with the command : `grafana-cli plugins install grafana-image-renderer`

```
nov. 30 16:21:42 serin grafana-server[63604]: t=2021-11-30T16:21:42+0100 lvl=eror msg="Stopped background service *rendering.RenderingService" logger=server reason="Failed to start renderer plugin: Unrecognized remote plugin message: \n\nThis usually means that the plugin is either invalid or simply\nneeds to be recompiled to support the latest protocol."
nov. 30 16:21:42 serin grafana-server[63604]: t=2021-11-30T16:21:42+0100 lvl=warn msg="plugin failed to exit gracefully" logger=plugins.backend pluginId=grafana-image-renderer
nov. 30 16:21:42 serin grafana-server[63604]: t=2021-11-30T16:21:42+0100 lvl=eror msg="Server shutdown" logger=server error="*rendering.RenderingService run error: Failed to start renderer plugin: Unrecognized remote plugin message: \n\nThis usually means that the plugin is either invalid or simply\nneeds to be recompiled to support the latest protocol."
nov. 30 16:21:42 serin grafana-server[63604]: *rendering.RenderingService run error: Failed to start renderer plugin: Unrecognized remote plugin message:
nov. 30 16:21:42 serin grafana-server[63604]: This usually means that the plugin is either invalid or simply
nov. 30 16:21:42 serin grafana-server[63604]: needs to be recompiled to support the latest protocol.
nov. 30 16:21:42 serin systemd[1]: grafana-server.service: Main process exited, code=exited, status=1/FAILURE
nov. 30 16:21:42 serin systemd[1]: grafana-server.service: Failed with result 'exit-code'.
nov. 30 16:21:42 serin systemd[1]: grafana-server.service: Consumed 1.413s CPU time.
nov. 30 16:21:42 serin systemd[1]: grafana-server.service: Scheduled restart job, restart counter is at 5.
nov. 30 16:21:42 serin systemd[1]: Stopped Grafana instance.
nov. 30 16:21:42 serin systemd[1]: grafana-server.service: Consumed 1.413s CPU time.
nov. 30 16:21:42 serin systemd[1]: grafana-server.service: Start request repeated too quickly.
nov. 30 16:21:42 serin systemd[1]: grafana-server.service: Failed with result 'exit-code'.
nov. 30 16:21:42 serin systemd[1]: Failed to start Grafana instance.
```

**What you expected to happen**:
On service reboot, I expected Grafana to come up properly without any issues

**How to reproduce it (as minimally and precisely as possible)**:
Upgrade Grafana to version 8.2.5 and install the Image Renderer plugin

**Anything else we need to know?**:
I've seen the issue #40162 and it was supposed to be fixed with the version 8.2.1 but it ain't working for me.

The workaround I found is to modify in `/usr/lib/systemd/system/grafana-server.service` the line `MemoryDenyWriteExecute` from `yes` to `false`. I didn't have to modify any `SystemCallFilter` lines.

**Environment**:
- Grafana version: 8.2.5
- OS Grafana is installed on: Debian 11.1
- Grafana plugins: Grafana Image Renderer
- Others:

**Comment by wbrowne** [comment]
Hi @Darktortue,

>I've seen the issue #40162 and it was supposed to be fixed with the version 8.2.1 but it ain't working for me.

Not sure how it's possible given the specific change to `MemoryDenyWriteExecute` was targeting `8.2.1` (see [here](https://github.com/grafana/grafana/pull/40176)). Are you sure you had a fresh install?

**Comment by marefr** [comment]
Closing this due to lack of response. We'll consider reopen the issue if you provide more information.
